flink-cdc-connect/flink-cdc-source-connectors/flink-connector-mysql-cdc/src/main/java/org/apache/flink/cdc/connectors/mysql/debezium/task/MySqlSnapshotSplitReadTask.java

```diff
@@ -109,6 +117,21 @@ public MySqlSnapshotSplitReadTask(
         this.snapshotChangeEventSourceMetrics = snapshotChangeEventSourceMetrics;
         this.hooks = hooks;
         this.isBackfillSkipped = isBackfillSkipped;
+
+        // setting column filter
+        String columnIncludeList =
+                sourceConfig.getDbzConfiguration().getString(COLUMN_INCLUDE_LIST);
+        String columnExcludeList =
+                sourceConfig.getDbzConfiguration().getString(COLUMN_EXCLUDE_LIST);
```

Comment:
It seems that only one of `COLUMN_INCLUDE_LIST` and `COLUMN_EXCLUDE_LIST` will take effect.
And `COLUMN_INCLUDE_LIST` has a higher priority.

Please add an warning for the useless setting.

Reply:
done